Add a libtrixi cmake module (#132)
trixi-framework/libtrixi: docs/src/index.md

```
to build `simple_trixi_controller_f`.

#### CMake

A CMake module for the discovery of an installed libtrixi library is provided with

> [`cmake/FindLibTrixi.cmake`](cmake/FindLibTrixi.cmake). Upon invocation of
> `find_package(LibTrixi)`, the CMake variable `LIBTRIXI_PREFIX` must be set to
> `<install_directory>`. An example `CMakeLists.txt` can be found in

Reviewer: ```suggestion
[`cmake/FindLibTrixi.cmake`](cmake/FindLibTrixi.cmake). Before calling
`find_package(LibTrixi)`, the CMake variable `LIBTRIXI_PREFIX` must be set to
`<install_directory>`. An example `CMakeLists.txt` can be found in
```

> [`examples/CMakeListsExternal.txt`](https://github.com/trixi-framework/libtrixi/tree/main/examples/CMakeListsExternal.txt).

Reviewer: ```suggestion
[`examples/external/CMakeLists.txt`](https://github.com/trixi-framework/libtrixi/tree/main/examples/external/CMakeLists.txt).
To see the commands required to build an example program with this CMake project,
please refer to
[`examples/external/build.sh`](https://github.com/trixi-framework/libtrixi/tree/main/examples/external/build.sh).
```
